### PR TITLE
[FEAT] Fresh application page and new avatar and fallbacks.

### PR DESCRIPTION
**Pull request by simonsasse**

### CODE REVIEW

File: _layouts/app.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h2>Links</h2>
 
     <h2>Please Cite</h2>
       {% for citation in page.cite %}
```
Comment (marehr) [inline _layouts/app.html:24]:
```suggestion
      <ul>
      {% for citation in page.cite %}
```

File: _layouts/app.html [hunk 2]
```diff
@@ -22,7 +22,7 @@ <h2>Links</h2>
...
-        <p>{%- include cite.html cite=citation -%}</p>
+        <ul><li>{%- include cite.html cite=citation -%}</li></ul>
       {% endfor %}
     <h2>Contact</h2>
```
Comment (marehr) [inline _layouts/app.html:27]:
```suggestion
        <li>{%- include cite.html cite=citation -%}</li>
      {% endfor %}
    </ul>
    <h2>Contact</h2>
```

File: _layouts/app.html [hunk 3]
```diff
@@ -21,9 +21,11 @@ <h2>Links</h2>
     </ul>
 
     <h2>Please Cite</h2>
+      <ul>
```
Comment (marehr) [inline _layouts/app.html:24]:
```suggestion
    <ul>
```

**Comment by simonsasse** [comment]
Could add the fallback if no citation found.

**Comment by marehr** [comment]
Yep, can you do this?

**Comment by simonsasse** [comment]
I'm on it

File: assets/images/people/avatar_dummy.svg [hunk 4]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (marehr) [inline assets/images/people/avatar_dummy.svg:1]: Can you add a `avatar_dummy.svg.source` file that links back to the source?

File: _layouts/app.html [hunk 5]
```diff
@@ -23,18 +23,36 @@ <h2>Links</h2>
       <li>View the source code and README on <a href="{{page.links.source}}">GitHub</a></li>
     </ul>
 
+    <!-- if no citation is given, the whole Block is skipped. -->
+    <!-- if give citation not found, cite.html will show an error message. -->
```
Comment (eseiler) [inline _layouts/app.html:27]:
```suggestion
    <!-- if given citation is not found, cite.html will show an error message. -->
```

File: _layouts/app.html [hunk 6]
```diff
@@ -23,18 +23,36 @@ <h2>Links</h2>
...
+      {% if author.avatar %}
+        <img class="person-avatar" src="{{author.avatar | relative_url}}">
+      {% else %}
+        <img class="person-avatar" src="/assets/images/people/avatar_dummy.svg">
+      {% endif %}
+      <span class="person-name-position">
+          <span class="person-name">{{ author.name }}</span>
+          <small class="person-position"><a href="mailto:{{author.social_links.email}}">{{author.social_links.email}}</a></small>
+      </span>
+    {% else %} <!-- if author not present, just print the name and the dummy avatar. -->
```
Comment (eseiler) [inline _layouts/app.html:50]:
```suggestion
    {% else %} <!-- if author is not present, just print the name and the dummy avatar. -->
```

File: assets/images/people/avatar_dummy.svg (hunk 4 above)
Comment (simonsasse) [reply assets/images/people/avatar_dummy.svg:1]: ... containing just the link ?

**Review by marehr — APPROVED** [review]

**Pull request merged**